src/lib/components/wizard.component-nested-steps.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, ViewChild } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ArchwizardModule } from '../archwizard.module';
+import { WizardComponent } from './wizard.component';
+
+@Component({
+    selector: 'aw-test-wizard',
+    template: `
+        <aw-wizard>
+            <div>
+                <aw-wizard-step></aw-wizard-step>
+            </div>
+            <div>
+                <aw-wizard-step></aw-wizard-step>
+            </div>
+            <div>
+                <div>
+                    <aw-wizard-step></aw-wizard-step>
+                </div>
+            </div>
+        </aw-wizard>
+  `
+})
+class WizardTestComponent {
+    @ViewChild(WizardComponent) public wizard: WizardComponent;
+}
+
+describe('WizardComponent', () => {
+    let wizardTestFixture: ComponentFixture<WizardTestComponent>;
+
+    let wizardTest: WizardTestComponent;
+    let wizard: WizardComponent;
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            declarations: [WizardTestComponent],
+            imports: [ArchwizardModule]
+        }).compileComponents();
+    }));
+
+    beforeEach(() => {
+        wizardTestFixture = TestBed.createComponent(WizardTestComponent);
+        wizardTestFixture.detectChanges();
+
+        wizardTest = wizardTestFixture.componentInstance;
+        wizard = wizardTest.wizard;
+    });
```

Comment:
Now that #264 has been merged these lines most likely needs to be changed slightly to make it work correctly:

https://github.com/madoar/angular-archwizard/blob/d3f4071451b638b5783f8842d167a7fba50f38ac/src/lib/components/wizard.component.spec.ts#L54-L64

Reply:
Formatting changed to prettier, changed included :)